plugins/janus_videoroom.c

```diff
@@ -4015,6 +4043,7 @@ static json_t *janus_videoroom_process_synchronous_request(janus_videoroom_sessi
 		json_object_set_new(response, "videoroom", json_string("created"));
 		json_object_set_new(response, "room", string_ids ? json_string(videoroom->room_id_str) : json_integer(videoroom->room_id));
 		json_object_set_new(response, "permanent", save ? json_true() : json_false());
+		if(dynamic_url_status) json_object_set_new(response, "sgwURL", json_string(videoroom->sgwURL));
```

Comment:
Wrap with usual {} and the json_object_set_new in a line of its own

Reply:
fixed